.github/workflows/release.yml

```
@@ -346,7 +333,8 @@ jobs:
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
 
-      - name: Publish the release
+      - name: Publish the release  # FIXME: Reenable this.
+        if: false
```

Before merging, actual publishing should be reenabled:

```suggestion
      - name: Publish the release
```

I have no objection to this being merged at any time with that change, but as noted above I suspect you may want other changes, so for now this is still marked as a draft.

---

Obsolete,  871eec7 includes this.